tests/test_version.py

```diff
@@ -11,7 +11,7 @@
 def test_version_info():
     s = version_info()
     assert re.match(' *pydantic version: ', s)
-    assert s.count('\n') == 6
+    assert s.count('\n') == 7
```

Comment:
probably we can check the existence of `commit`. `assert re.match(' *commit: ', s)`

Reply:
Yeah. I suppose we should do this for the other fields as well.